mpas_analysis/ocean/nino34_timeseries.py

```diff
@@ -0,0 +1,198 @@
+from ..shared.plot.plotting import nino34_timeseries_plot, nino34_spectra_plot
```

Comment:
Do you want to have a module header docstring block? For example, this would be something like
```python
"""
Computes NINO34 diagnostic.

Author: Luke Van Roekel
Last Modified: 03/21/2017
"""
```